src/hb-ot-color-colr-table.hh

```diff
@@ -90,6 +90,474 @@ struct BaseGlyphRecord
   DEFINE_SIZE_STATIC (6);
 };
 
+template <typename T>
+struct Variable
+{
+  bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
+
+  protected:
+  T      value;
+  VarIdx varIdx; /* Use 0xFFFFFFFF to indicate no variation. */
```

Comment:
The comment is unnecessary. Belongs to where VarIdx is declared.  But then again, it is there already:
```
struct VarIdx : HBUINT32 {
  static constexpr unsigned NO_VARIATION = 0xFFFFFFFFu;
  VarIdx& operator = (uint32_t i) { HBUINT32::operator= (i); return *this; }
};
```